index.html

```diff
@@ -3,6 +3,10 @@
     <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
     <title>JS fun!</title>
     <script type="text/javascript" src="main.js"></script>
+	<style> 
+	h1
+	{ color: blue}
+	</style>
```

Comment:
I tested it, it works. 

![01](https://user-images.githubusercontent.com/13213915/37554527-9e3a5e94-29da-11e8-84f9-84900e6bb522.png)